pkg/ttl/ttlworker/timer_test.go

```diff
@@ -350,8 +351,9 @@ func TestTTLTimerHookOnEvent(t *testing.T) {
 	require.Equal(t, summaryData, timer.SummaryData)
 	adapter.AssertExpectations(t)
 
+	tz := timeutil.SystemLocation()
 	// job not exists but table ttl not enabled
-	watermark := time.Unix(3600*123, 0)
+	watermark := time.Unix(3600*123, 0).In(tz)
```

Comment:
A fix to make this test stable in CI environment (when the `SystemLocation` is possibly not `Local`). I've faced the same error in https://github.com/pingcap/tidb/pull/57718, so I think it's not related to other changes in this PR.